### PR TITLE
Feature base django app

### PR DESCRIPTION
**Pull request by maciejSamerdak**
An updated and renamed 'project-initialization' branch featuring requested changes from the last pull request review.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Django>2.1, <2.2
```
Comment (kbeker) [inline requirements.txt]: It should be `Django =>, <2.2`

File: time_monkey/settings/base.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
```
Comment (kbeker) [inline time_monkey/settings/base.py]: You should not delete `PRIV_KEY` at all from here. Add here, but as a comment and add description what is it used for.

File: time_monkey/settings/base.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+# Database
+# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE':           'django.db.backends.postgresql',
+        'NAME':             'time_monkey',
+        'ATOMIC_REQUESTS':  True,
+    }
```
Comment (kbeker) [inline time_monkey/settings/base.py]: Please also add here 
```python
# 'USER':     'postgres',
# 'PASSWORD': '',
# 'HOST':     '',
# 'PORT':     '',
```
It should be added as a comment

Comment (maciejSamerdak) [reply time_monkey/settings/base.py]: Do you mean inside the bracket, below the previous fields, but commented out for future use i.e. 'the intern way'

or

above the code, as a template with perhaps additional information regarding it's usage?

File: requirements.txt (hunk 1 above)
Comment (maciejSamerdak) [reply requirements.txt]: So this way it refers directly to the version specified in the .lock file, correct?

Comment (kbeker) [reply requirements.txt]: Sorry, I see that I have not write it correctly last time. It should be `Django => 2.1, < 2.2`

File: time_monkey/settings/base.py (hunk 3 above)
Comment (kbeker) [reply time_monkey/settings/base.py]: Just inside the bracket :)

**Pull request merged**